docs/user-guide/recipes.rst

```diff
@@ -354,3 +354,66 @@ This produces a tarball at
 both a JSON file for use with crane, and the static Image files to which crane
 will redirect requests. See the `Crane`_ documentation for how to use that
 tarball.
+
+Tagging a Manifest
+------------------
+
+Using the docker repo tag command, we can point a docker tag to a manifest. If
+the tag we specify does not exist, it will be created. If the tag exists
+however, it will be updated as tag name is unique per repository and can point
+to only one manifest.
+
+For instance, suppose we have the following manifests::
+
+    $ pulp-admin docker repo search tag --repo-id busybox
```

Comment:
i don't think this is what ` docker repo search tag` command displays, but `docker repo seach manifest`;
please update docs with correct info